### PR TITLE
Various doc improvements

### PR DESCRIPTION
**Pull request by axelson**
/data should be used instead of /root

Add a link to info about pushing updates remotely to the getting started
guide

Add notes to FAQ:
- Where to store persistent data
- Applying a firmware update manually (command needs tweaking still)
- Pushing firmware updates remotely

### CODE REVIEW

File: docs/FAQ.md [hunk 1]
```diff
@@ -4,6 +4,23 @@ This is a collection of questions that often come up as people are getting start
...
 
+## Where can persistent data be stored?
+
+By default the `/data` partition both read/writeable and is not overwritten when new firmware is pushed to the device.
+
+## How can I apply a firmware update manually?
+
+```
+iex> cmd("fwup -i /data/known_good.fw --apply --task complete " <>
+  "--no-unmount -d #{Nerves.Runtime.KV.get("nerves_fw_devpath")}")
```
Comment (axelson) [inline docs/FAQ.md:15]: I built this command by looking at what https://github.com/nerves-project/nerves_firmware_ssh/blob/86e43429c87544c7d18a478bfd200df5667466ff/lib/nerves_firmware_ssh/fwup.ex does. However, this command does not work as-is (testing so far on nerves_system_vultr). On boot I get the following errors:
```
19:49:53.921 [error] FAT-fs (vda1): codepage cp437 not found
19:49:53.921 [warn]  erlinit: Cannot mount /dev/vda1 at /mnt: Invalid argument
19:49:53.921 [error] EXT4-fs (vda4): ext4_check_descriptors: Block bitmap for group 0 not in group (block 4294967295)!
19:49:53.921 [error] EXT4-fs (vda4): group descriptors corrupted!
19:49:53.921 [warn]  erlinit: Cannot mount /dev/vda4 at /root: No error information
19:49:53.924 [info]  mount: mounting /dev/vda4 on /root failed: No error information
19:49:53.925 [warn]  heart: kernel watchdog activated (interval 5s)
19:49:53.925 [error] EXT4-fs (vda4): ext4_check_descriptors: Block bitmap for group 0 not in group (block 4294967295)!
19:49:53.925 [error] EXT4-fs (vda4): group descriptors corrupted!
19:49:53.925 [warn]  Ignoring non-zero exit status from mount ["-t", "ext4", "-o", "rw", "/dev/vda4", "/root"]
```

Also, I believe we'll need to tell the user to run `reboot/0`.

Comment (fhunleth) [reply docs/FAQ.md:15]: If you change the task to "upgrade", it should work.

Comment (axelson) [reply docs/FAQ.md:15]: Ah, that does work :+1:. Didn't expect "upgrade" to be a task based on the `fwup -l` output:
```
fwup -l -i good.fw
complete
upgrade.a
upgrade.b
upgrade.unexpected
upgrade.wrongplatform
```
I thought you would need to do `upgrade.a` or b

**Comment by axelson** [comment]
Okay, made some slight updates to this, and changed the task to "upgrade"

File: docs/FAQ.md (hunk 1 above)
Comment (fhunleth) [reply docs/FAQ.md:17]: Yeah, the task passed on the commandline is a prefix match for the first applicable task in the fwup.conf file. That lets you force fwup to upgrade the A or B partition or let fwup decide which is the right one.

**Comment by fhunleth** [comment]
Looks good! Thanks.

File: docs/FAQ.md (hunk 1 above)
Comment (axelson) [reply docs/FAQ.md:17]: Okay, that makes sense :+1:

**Pull request merged**